docs/integrating-into-custom-elements.md

```diff
@@ -0,0 +1,61 @@
+# Integrating MDLv2 into custom elements
```

Comment:
This is a good start, but I'm not sure about the approach of having a doc for this, where we teach folks to do it themselves. It seems like a halfway solution, where instead we'd want to distribute optional support for custom elements together with our packages, or with a set of mirror packages. This could potentially be as easy as making the user add another `import` statement, and we could even mark it as highly experimental.

That said, I understand us not wanting to do it right now (particularly just before alpha), but maybe we want to reflect that offering built-in support is something we're considering for the future?